e2e/chat.spec.ts

```diff
@@ -384,6 +384,12 @@ describe('chat', () => {
     const chat = new Chat(page);
 
     await page.addInitScript(() => {
+      const systemPrompt = {
+        role: 'system',
+        content:
+          'You are an AI assistant knowledgeable about blockchain technologies.',
+      };
+
```

Comment:
Add a system prompt to the mock conversation because we need to ignore that when searching content. Note that it doesn't occur as the match when the user searches "blockchain."